website/docs/scaffolds/nft-scaffold.mdx

```diff
@@ -15,6 +15,14 @@ title: The NFT Scaffold Project
 The NFT-Scaffold project is currently in alpha. It is not ready for production use and will be undergoing frequent updates as we work to implement Taqueria State and other features that have not yet been released
 :::
 
+## Requirements
+
+- Taqueria v0.16.0 or later
+- Docker v0.9 or later
+- Node.js v16 or later
```

Comment:
Should we specify NodeJS v16.17 or later instead?

Reply:
Definitely